client/src/annoMatrix/schema.ts

```diff
@@ -36,8 +36,7 @@ export function _getColumnSchema(
 
 export function _isIndex(schema: Schema, field: Field, col: string): boolean {
   // @ts-expect-error ts-migrate --- TODO revisit:
-  // Compiler is complaining that Field needs to be narrowed to the appropriate values (that is, obs or var and not
-  // emb and X). Possibly add field value validation here and throw on invalid values.
+  // `schema.annotations?.[field]`: Element implicitly has an 'any' type because expression of type 'Field' can't be used to index type 'AnnotationsSchema'. Property '[Field.emb]' does not exist on type 'AnnotationsSchema'.
```

Comment:
Why not fix the schema interface to allow indexing by Field enum?  If we are going to define Field, we might as well use it to catch errors (eg, `annotations['0bs']` should generate an error!)

Reply:
Updated [schema.ts:_isIndex](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/schema.ts#L40).